src/main/app/utils/keywordHearingEventUtils.ts

```diff
@@ -0,0 +1,23 @@
+import { listHearingKeywords } from './listHearingKeywords'
+
+export class KeywordHearingEventUtils {
+  static getKeywordHearingEvent (amount: number) {
+    let keyword: string = ''
+
+    if (amount >= 0.01 && amount <= 300) {
+      keyword = listHearingKeywords[0]
```

Comment:
relying on the index of array does not seems to be nice here, as if we add/remove anything in array this while logic will break. Let's use constants/enum for each instead